server/user.go

```diff
@@ -92,7 +94,7 @@ func (s *UserMgmtServer) authAPIHandle(handle authedHandle, requiresAdmin bool)
 			s.writeError(w, err)
 			return
 		}
-		if creds.User.Disabled || (requiresAdmin && !creds.User.Admin) {
+		if creds.User.ID != "" && (creds.User.Disabled || (requiresAdmin && !creds.User.Admin)) {
```

Comment:
`creds.User.ID != ""` shouldn't be checked here. There should be two paths for this code.

```
if clientcred {

} else {

}
```